internal/service/project/data_source_project_test.go

```diff
@@ -80,8 +83,9 @@ func TestAccProjectDSProject_byName(t *testing.T) {
 
 func TestAccProjectDSProject_defaultFlags(t *testing.T) {
 	var (
-		projectName = acctest.RandomWithPrefix("test-acc")
-		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName    = acctest.RandomWithPrefix("test-acc")
+		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		dataSourceName = "data.mongodbatlas_project.test"
```

Comment:
knitpit: as this is always the same, in other resources I put dataSourceName (and resourceName) in a const block in the file so it's not repeat in each test

Reply:
like this idea, adjusted.